internal/plugin/plugin.go

```diff
@@ -246,15 +266,20 @@ func createSymlink(root, filePath string) error {
 	return errors.WithStack(os.Symlink(filePath, newname))
 }
 
-func (m *Manager) shouldCreateFile(pkg *lock.Package, filePath string) bool {
+func (m *Manager) shouldCreateFile(
+	pkg *lock.Package,
+	filePath,
+	virtenvPath string,
+) bool {
 	// Only create files in devboxDir if they are not in the lockfile
 	pluginInstalled := pkg != nil && pkg.PluginVersion != ""
 	if strings.Contains(filePath, devboxDirName) && pluginInstalled {
 		return false
 	}
 
 	// Hidden .devbox files are always replaceable, so ok to recreate
-	if strings.Contains(filePath, devboxHiddenDirName) {
+	if strings.Contains(filePath, devboxHiddenDirName) ||
+		strings.HasPrefix(filePath, virtenvPath) {
```

Comment:
Will these always be relative paths for `HasPrefix`?

Reply:
Both `filePath` and `virtenvPath` should be absolute.